frontend/src/metabase/visualizations/echarts/cartesian/model/dataset.unit.spec.ts

```diff
@@ -178,4 +186,59 @@ describe("dataset transform functions", () => {
       expect(result).toEqual([]);
     });
   });
+
+  describe("replaceValues", () => {
+    it("should replace missing values with zeros according to the replacer function", () => {
```

Comment:
I would make this more generic to make it clear that this is independent from the `getNullReplacerFunction` E.g. `it("should replace missing values according to the replacer function"`

Then in the test you can make the `replacer` function use a different value then `0`, e.g. a constant string `"replacement"`.